tests/RobotFramework/tests/MQTT_health_check/MQTT_health_endpoints.robot

```diff
@@ -16,17 +16,17 @@ tedge-log-plugin health status
 
     Sleep    5s    reason=It fails without this! It needs a better way of queuing requests
     ${pid}=    Execute Command    pgrep -f '^/usr/bin/tedge-log-plugin'    strip=${True}
-    Execute Command    sudo tedge mqtt pub 'tedge/health-check/tedge-log-plugin' ''
+    Execute Command    sudo tedge mqtt pub 'te/device/main/service/tedge-log-plugin/cmd/health/check' ''
     ${messages}=    Should Have MQTT Messages    te/device/main/service/tedge-log-plugin/status/health    minimum=1    maximum=2
     Should Contain    ${messages[0]}    "pid":${pid}
     Should Contain    ${messages[0]}    "status":"up"
 
 c8y-configuration-plugin health status
     Execute Command    sudo systemctl start c8y-configuration-plugin.service
-   
-    Sleep    5s     reason=It fails without this! It needs a better way of queuing requests
-    ${pid}=    Execute Command    pgrep -f '^/usr/bin/c8y[_-]configuration[_-]plugin'    strip=${True}
-    Execute Command    sudo tedge mqtt pub 'tedge/health-check/c8y-configuration-plugin' ''
-    ${messages}=    Should Have MQTT Messages    te/device/main/service/c8y-configuration-plugin/status/health    minimum=1    maximum=2
+
+    Sleep             5s                 reason=It fails without this! It needs a better way of queuing requests
```

Comment:
Your IDE settings probably need some tweaks as it looks like it's trying to align the arguments of all function calls. We just follow a simpler style with just "4 spaced tabs". May be it's time we officially enabled some linter for Python/Robot code as well so that such issues can be avoided.